tanner/emulators/sqli.py

```diff
@@ -70,9 +70,17 @@ def map_query(self, query):
         db_query = None
         param = query[0][0]
         param_value = query[0][1].replace('\'', ' ')
-        tables = [k for k, v in self.query_map.items() if query[0][0] in v]
+        tables = []
+        for table, columns in self.query_map.items():
+            for column in columns: 
+                if query[0][0] == column['name']:
```

Comment:
I've changed that in `db_helper.py`, now it is a `dict`. Please take a look in db_helper.py.